src/DeviceInterface.js

```diff
@@ -17,9 +19,9 @@ function createDevice () {
      */
     const loggingTransport = {
         async send (deviceApiCall) {
-            console.log('[outgoing]', deviceApiCall.method, 'id:', deviceApiCall.id, JSON.stringify(deviceApiCall.params || null))
+            console.log('[->outgoing]', 'id:', deviceApiCall.method, JSON.stringify(deviceApiCall.params || null))
             const result = await transport.send(deviceApiCall)
-            console.log('[incoming]', deviceApiCall.method, 'id:', deviceApiCall.id, JSON.stringify(result || null))
+            console.log('[<-incoming]', 'id:', deviceApiCall.method, JSON.stringify(result || null))
```

Comment:
yeah that's a good point - perhaps we can follow up with some debugging improvements, especially since you've had some other comments on this recently :)